src/value.rs

```diff
@@ -135,8 +218,14 @@ pub trait ValueType: Sized {
         Self::try_from(v).unwrap()
     }
 
+    fn except(v: Value, msg: &str) -> Self {
+        Self::try_from(v).expect(msg)
+    }
```

Comment:
Would it be better to rename it as `expect`? Since it's a wrapper method of `try_from` then `expect`.

This also align with the API of... few lines above it
```rs
fn unwrap(v: Value) -> Self {
    Self::try_from(v).unwrap()
}
```

Reply:
Fix!